wp-includes/functions.php

```diff
@@ -5736,19 +5736,36 @@ function wp_validate_boolean( $var ) {
 /**
  * Delete a file
  *
- * @since 4.2.0
+ * @since 4.9.2
```

Comment:
The function has been in WP since `4.2.0`. That hasn't changed.
What you want to do instead is add a second `@since` entry to annotate what has changed, like so:

```php
* @since 4.2.0
* @since 4.9.2 Added the `$directory_unblock` parameter.
```

For more information, have a look at the [WP PHP Documentation standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/).

Reply:
Done.